config/v1/types_cluster_operator.go

```diff
@@ -176,10 +176,15 @@ const (
 	// unexpected errors are handled as operators mature.
 	OperatorDegraded ClusterStatusConditionType = "Degraded"
 
-	// Upgradeable indicates whether the operator safe to upgrade based on the current cluster state. When status is `False`
-	// administrators should not upgrade their cluster and the message field should contain a human readable description
-	// of what the administrator should do to allow the operator to successfully update.  A missing condition, True,
-	// and Unknown are all treated by the CVO as allowing an upgrade.
+	// Upgradeable indicates whether the operator is safe to upgrade based on the
+	// current cluster state. When status is False, the cluster-version operator
+	// will prevent the cluster from performing impacted updates unless forced.
+	// When set on ClusterVersion, the message will explain which updates are
+	// impacted.  When set on ClusterOperator, False will block minor OpenShift
+	// updates.  The message field should contain a human readable description
+	// of what the administrator should do to allow the cluster or operator to
+	// successfully update.  A missing condition, True, and Unknown are all
+	// treated by the cluster-version operator as allowing an upgrade.
```

Comment:
s/A missing condition, True, and Unknown are all treated by the cluster-version operator as allowing an upgrade/`cluster-version operator will allow the upgrade for missing , true and unknown condition`/

Reply:
> cluster-version operator
>	// will prevent the cluster from performing impacted updates unless forced.

based on what i heard this morning, this isn't true, the CVO doesn't prevent it, only the client tools prevent you from targeting the upgrade, but if you set the target manually via the api, the CVO will happily do the upgrade.

(we need to evaluate if that is the *right* behavior, but that's what i understood it to be from @sdodson )

Reply:
I may have been wrong, hopefully Trevor can correct me if so and we can make sure there's a concise summary of all the inputs to the CVOs decision to accept, reject, or halt an upgrade.

Reply:
In 5de73edb -> 7ef0a538, I've shuffled to make the CVO the subject, per your recommendation, but with slightly different wording later:

> The cluster-version operator will allow updates when this condition is not False, including when it is missing, True, or Unknown.

I will optimistically mark this thread resolved, but reopen or commend on the new wording if you want further edits.